webapp/app.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+
+from containers.containers import containers_bp
+from general.general import general_bp
+
+app = Flask(__name__)
+
+app.config.from_object('config.ProdConfig')
+
+app.register_blueprint(general_bp)
+app.register_blueprint(containers_bp, url_prefix='/containers')
+
+if __name__ == '__main__':
```

Comment:
Can we add some exceptions processing? What if some of this methods will rise an exception? I thinks, we have to be able process it to close application correct.

Reply:
Resolved